common.blocks/menu-item/menu-item.spec.js

```diff
@@ -37,6 +39,26 @@ describe('menu-item', function() {
         });
     });
 
+    describe('disabled', function() {
+        it('should set "aria-disabled" attribute if disabled', function() {
+            menuItem.setMod('disabled', true);
+            expect(menuItem.domElem.attr('aria-disabled')).to.equal('true');
```

Comment:
Please be consistence! At least within one file (e.g. look how the code have been written several lines below):

1. no need this `expect()` at this particular case;
2. we use "should"-style of BDD, so should be `menuItem.domElem.attr('aria-disabled').should.be.equal('true')`